employee-management/interface-document.md

```diff
@@ -311,43 +368,27 @@
 - HTTP status
     - 200 OK
     - 404 Not Found
-
-#### Body
-
-- Cập nhập thông tin nhân viên từ id được chỉ định
-
-##### Sample response
-
-```json
-{
-  "id": "123",
-  "first_name": "bien",    
-  "last_name": "danghuu",
-  "date_of_birth": "1996-02-15",
-  "salary": 50000
-}
-```
-
+    
 #### Error response
 
 ##### Parameter error
 
-|HTTP Status|Error code|Error message|Description |
+|HTTP Status|Error code|Error message|Description|
 |:----|:----|:----|:----|
-|400|invalid_id|Employee ID is invalid.|The requested Employee ID is contains Non-Nummeric characters. |
-|400|invalid_date_of_birth|Date Of Birth is invalid.|The requested Employee's Birthday contains non-string characters. |
-|400|invalid_salary|Request employee salary is invalid.|The requested Employee salary is contains non-string characters. |
-|400|required_id|Must not be empty.|Return if employee id is not specified. |
-|404|not_found|Not found employee.|Not found employee information. |
+|400|required_id|Must not be empty.|Employee id is not specified.|
+|400|invalid_employee_id|Employee ID is invalid.|The requested Employee ID is contains Non-Nummeric characters.|
+|400|invalid_date_of_birth|Date of birth is invalid.|The user enters a birthday that is not a valid date format.|
```

Comment:
`The user ` là thằng quái nào vậy? linh tinh

Reply:
cũng là description cho error đối với date format, mà lại ghi mỗi nơi mỗi khác?

Reply:
Vâng em đã chỉnh sửa lại.